vsintegration/src/FSharp.Editor/Classification/ColorizationService.fs

```diff
@@ -35,7 +35,7 @@ type internal FSharpColorizationService
 
         member this.AddSemanticClassificationsAsync(document: Document, textSpan: TextSpan, result: List<ClassifiedSpan>, cancellationToken: CancellationToken) =
             asyncMaybe {
-                let! options = projectInfoManager.TryGetOptionsForEditingDocumentOrProject(document)
+                let! options = projectInfoManager.TryGetOptionsForDocumentOrProject(document)
```

Comment:
This is much slower isn't it?  The point about ``TryGetOptionsForEditingDocumentOrProject`` is to be fast as a tradeoff.